npm-scripts.js

```diff
@@ -17,9 +17,19 @@ const MAKE = process.env.MAKE || (isFreeBSD ? 'gmake' : 'make');
 // eslint-disable-next-line no-console
 console.log(`npm-scripts.js [INFO] running task "${task}"`);
 
-if (isWindows && fs.existsSync('worker/out/msys/bin/make.exe'))
+if (isWindows)
 {
-	addMsysToPath();
+	if (!fs.existsSync('worker/out/msys/bin/make.exe'))
+	{
+		installMsysMake();
+	}
+
+	const msysPath = `${process.cwd()}\\worker\\out\\msys\\bin`;
+
+	if (process.env['PATH'].indexOf(msysPath) < 0)
```

Comment:
Better this:
```suggestion
	if (!process.env['PATH'].includes(msysPath))
```

However I'm not sure the check is even necessary, it doesn't hurt even if included multiple times.